test-ts/manager.test.ts

```diff
@@ -1200,4 +1218,164 @@ describe("Manager", () => {
       });
     });
   });
+
+  describe("#setDependencies()", () => {
+    let ownerSigner: SignerWithAddress;
+
+    before(async () => {
+      const managerOwner = await manager.owner();
+      ownerSigner = await getImpersonatedSigner(managerOwner);
+    });
+
+    it("reverts with zero stCelo address", async () => {
+      await expect(
+        manager
+          .connect(ownerSigner)
+          .setDependencies(ADDRESS_ZERO, nonAccount.address, nonVote.address)
+      ).revertedWith("stakedCelo null address");
+    });
+
+    it("reverts with zero account address", async () => {
+      await expect(
+        manager
+          .connect(ownerSigner)
+          .setDependencies(nonStakedCelo.address, ADDRESS_ZERO, nonVote.address)
+      ).revertedWith("account null address");
+    });
+
+    it("reverts with zero vote address", async () => {
+      await expect(
+        manager
+          .connect(ownerSigner)
+          .setDependencies(nonStakedCelo.address, nonAccount.address, ADDRESS_ZERO)
+      ).revertedWith("vote null address");
+    });
+
+    it("sets the vote contract", async () => {
+      await manager
+        .connect(ownerSigner)
+        .setDependencies(nonStakedCelo.address, nonAccount.address, nonVote.address);
+      const newVoteContract = await manager.voteContract();
+      expect(newVoteContract).to.eq(nonVote.address);
+    });
+
+    it("emits a VoteContractSet event", async () => {
+      const managerOwner = await manager.owner();
+      const ownerSigner = await getImpersonatedSigner(managerOwner);
+
+      await expect(
+        manager
+          .connect(ownerSigner)
+          .setDependencies(nonStakedCelo.address, nonAccount.address, nonVote.address)
+      )
+        .to.emit(manager, "VoteContractSet")
+        .withArgs(nonVote.address);
+    });
+
+    it("cannot be called by a non-Owner account", async () => {
+      await expect(
+        manager
+          .connect(nonOwner)
+          .setDependencies(nonStakedCelo.address, nonAccount.address, nonVote.address)
+      ).revertedWith("Ownable: caller is not the owner");
+    });
+  });
+
+  describe("#voteProposal()", () => {
+    const proposalId = 1;
+    const index = 0;
+    const yes = 10;
+    const no = 20;
+    const abstain = 30;
+
+    it("should call all subsequent contracts correctly", async () => {
+      const stakedCeloBalance = "1111";
```

Comment:
ditto: unused

Reply:
I introduces eslint into our project so this will not happen again :-)